app/Slack/Integration.php

```diff
@@ -6,8 +6,17 @@
  * @property int $id {primary}
  * @property string $name
  * @property string $hookUrl
+ * @property string|null $channel
  */
 class Integration extends \Nextras\Orm\Entity\Entity
 {
 
+	private const DEFAULT_CHANNEL = '#monitoring';
+
+
+	public function getChannel(): string
```

Comment:
`getterChannel`, ať je property nastavená vždy, což chceme. S normálním getterm bych na to musel myslet, že tam může být prázdný string třeba.

Reply:
Nextras ORM moc neznam, o tomhle jsem nevedel, upravim! :)